org.texmacs.TeXmacs.yml

```diff
@@ -0,0 +1,116 @@
+app-id: org.texmacs.TeXmacs
+runtime: org.kde.Platform
+runtime-version: "5.15-23.08"
+sdk: org.kde.Sdk
+command: texmacs
+rename-desktop-file: texmacs.desktop
+rename-icon: texmacs
+finish-args:
+  - --socket=x11
+  - --device=dri
+  - --share=ipc
+  - --filesystem=xdg-documents
+  - --persist=.TeXmacs
+  - --env=PATH=/app/texlive/bin/:/app/texlive/bin/x86_64-linux:/app/texlive/bin/aarch64-linux:/app/bin:/usr/bin
+  - --env=ASPELL_CONF=dict-dir /app/share/dicts
```

Comment:
This is not a valid env var

Reply:
Yes, I thought that, but it works. The runtime has support for Huspell and has Aspell installed but does not have dictionaries. If not this way I wouldn't know how.

Reply:
You should put it under quotes then this weird.